lib/nice_partials/monkey_patch.rb

```diff
@@ -46,13 +46,21 @@ def p(*args)
     end
   end
 
+  # Overrides `ActionView::Helpers::RenderingHelper#render` to push a new `nice_partial`
+  # on the stack, so rendering has a fresh `partial` to store content in.
   def render(*)
     __partials.prepend nice_partial
     super
   ensure
     __partials.shift
   end
 
+  # Action View passes any `yield`s in partials through `_layout_for`, our
+  # override detects if it's a capturing yield and appends the current partial
```

Comment:
This may be a nit pick, but consider if "our override" could more directly describe what the override is that is being referred to here. Is this something the user has created and is being referred to earlier in the docs? It may be just that I'm reading this narrow slice and don't have earlier context that could clue me in.

Reply:
@andyatkinson "our override" refers to the `_layout_for` that's being overridden here. Maybe this is clearer?

```suggestion
    # Action View passes any `yield`s in partials through `_layout_for`, so we
    # override `_layout_for` to detect if it's a capturing yield and appends the current partial
```

Reply:
That looks more clear to me @kaspth